### PR TITLE
[APP-SEC] Adding Terraform Module Scanning to Monitor and Manage Code Build

### PR DESCRIPTION
**Pull request by JBakstPaloAlto**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--prisma-cloud-docs--hlxsites.hlx.page/
- After: https://<branch>--prisma-cloud-docs--hlxsites.hlx.page/
- Worker: https://prisma-cloud-docs-production.adobeaem.workers.dev/?branch=<branch>

### CODE REVIEW

**Comment by JBakstPaloAlto** [comment]
Jira Ticket : https://redlock.atlassian.net/browse/RLP-125898

**Comment by manukumar6** [comment]
@JBakstPaloAlto, please let me know if we need to publish this earlier than 24.1.1.

File: docs/en/enterprise-edition/content-collections/application-security/risk-management/monitor-and-manage-code-build/terraform-module-scan.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+== Terraform Module Scanning
+
+Prisma Cloud supports rendering and scanning both public, private, and locally cached Terraform modules (modules). This capability enables users to analyze misconfigurations that may result from the module definition and the variables defined in the module block.
```
Comment (tsmithv11) [inline docs/en/enterprise-edition/content-collections/application-security/risk-management/monitor-and-manage-code-build/terraform-module-scan.adoc:3]:
```suggestion
Prisma Cloud supports rendering and scanning public, private, and locally cached Terraform modules (modules). This capability enables users to analyze misconfigurations that may result from the module definition and the variables defined in the module block.
```

**Review by tsmithv11 — APPROVED** [review]
LGTM

**Comment by manukumar6** [comment]
@JBakstPaloAlto, merged as requested.

File: docs/en/enterprise-edition/content-collections/application-security/risk-management/monitor-and-manage-code-build/terraform-module-scan.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+image::application-security/tf-module-scan.png[]
+
+=== Public and Locally Cached Modules
+
+Automated support for public modules is provided by the platform for all version control system (VCS) repositories that have been onboarded (For information on onboarding repositories refer to xref:../../get-started/connect-code-and-build-providers/code-repositories/code-repositories.adoc[Code Repositories]).
```
Comment (jenjoe22) [inline docs/en/enterprise-edition/content-collections/application-security/risk-management/monitor-and-manage-code-build/terraform-module-scan.adoc:9]: Automated support for public modules is provided by the platform for all Version Control System (VCS) repositories that have been onboarded. For information on onboarding repositories refer to xref:../../get-started/connect-code-and-build-providers/code-repositories/code-repositories.adoc[Code Repositories].

File: docs/en/enterprise-edition/content-collections/application-security/risk-management/monitor-and-manage-code-build/terraform-module-scan.adoc [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+==== Public and Local Module Scanning through Checkov CLI
+
+When using the Checkov CLI (CLI), make sure to include the `--download-external-modules true` flag to enable the automatic download of external modules.
+
+Additionally, the CLI supports scanning the `.terraform` directory when modules have been downloaded locally. Note that nested modules are not yet supported in scans performed in the  `.terraform` directory.
```
Comment (jenjoe22) [inline docs/en/enterprise-edition/content-collections/application-security/risk-management/monitor-and-manage-code-build/terraform-module-scan.adoc:15]: The last sentence should be a stand-alone Note.

File: docs/en/enterprise-edition/content-collections/application-security/risk-management/monitor-and-manage-code-build/terraform-module-scan.adoc (hunk 2 above)
Comment (JBakstPaloAlto) [reply docs/en/enterprise-edition/content-collections/application-security/risk-management/monitor-and-manage-code-build/terraform-module-scan.adoc:9]: 'version control systems': it is common to use lowercase letters, as it is a generic term. For example: "Git is a version control system"

File: docs/en/enterprise-edition/content-collections/application-security/risk-management/monitor-and-manage-code-build/terraform-module-scan.adoc (hunk 3 above)
Comment (JBakstPaloAlto) [reply docs/en/enterprise-edition/content-collections/application-security/risk-management/monitor-and-manage-code-build/terraform-module-scan.adoc:15]: Done

**Pull request merged**